codespell_lib/data/dictionary.txt

```diff
@@ -4620,6 +4645,8 @@ cicruit->circuit
 cicruits->circuits
 cicular->circular
 ciculars->circulars
+cilent->client
+cilents->clients
 cilincer->cylinder
 cilincers->cylinders
```

Comment:
I know it's not your change, but would you mind adding `silencer` and `silencers` to these too please?